contracts/stacks-abi-transformer/src/msg.rs

```diff
@@ -0,0 +1,24 @@
+use cosmwasm_schema::{cw_serde, QueryResponses};
+use cosmwasm_std::HexBinary;
+
+#[cw_serde]
+pub struct InstantiateMsg {}
+
+#[cw_serde]
+pub enum ExecuteMsg {}
+
+#[cw_serde]
+pub struct DecodeResponse {
+    pub clarity_payload: Vec<u8>,
+    pub payload_hash: [u8; 32],
+}
+
+#[cw_serde]
+#[derive(QueryResponses)]
+pub enum QueryMsg {
+    #[returns(DecodeResponse)]
+    DecodeReceiveFromHub { abi_payload: HexBinary },
+
+    #[returns(DecodeResponse)]
+    DecodeSendToHub { abi_payload: HexBinary },
```

Comment:
we need both directions here, and let's change the interface a bit:
```
     #[returns(interchain_token_service::Message)]
     FromBytes { payload: HexBinary},

     #[returns(HexBinary)]
     ToBytes { message: interchain_token_service::Message }
```
so the hub will call `ToBytes` when sending a message to stacks, and `FromBytes` when receiving a message from stacks. You will just call `FromBytes` in the current implementation, until we support this in the hub.

Reply:
For now we don't need to convert from clarity -> abi in the contract, we only need to do that in the Relayer.

I would refrain from implementing new functionality until ITS Hub supports payload conversions.

Reply:
we are trying to move this conversion to ITS hub as soon as possible, please get started on this

Reply:
Btw, the translator will need to encode/decode to/from `interchain_token_service::HubMessage` instead of the inner ITS message type